pkg/controller/datavolume-controller.go

```diff
@@ -508,12 +511,20 @@ func (r *DatavolumeReconciler) reconcileClone(log logr.Logger,
 				}
 				if !csiDriverAvailable {
 					// err csi clone not possible
+					storageClass, err := GetStorageClassByName(r.client, pvcSpec.StorageClassName)
+					if err != nil {
+						return reconcile.Result{}, err
+					}
+					if storageClass == nil {
+						log.Info("Could not find a default storage class in the cluster")
```

Comment:
It might be better to remove the word "default" and add the SC name

Reply:
yep, changed it to adjust the event message according to if we have the sc name or not